tests/foreman/cli/test_setting.py

```diff
@@ -245,10 +263,20 @@ def test_positive_update_email_subject_prefix(self):
 
         :expectedresults: email_subject_prefix is updated
 
-        :caseautomation: notautomated
+        :caseautomation: automated
 
         :caseimportance: low
         """
+        email_subject_prefix_value = gen_string('alpha')
+        Settings.set({
+            'name': "email_subject_prefix",
+            'value': email_subject_prefix_value
+        })
+        email_subject_prefix = Settings.list({
+            'search': 'name=email_subject_prefix'
+        })[0]
+        self.assertEqual(email_subject_prefix_value,
```

Comment:
```
self.assertEqual(
    email_subject_prefix_value,
    email_subject_prefix['value']
)
```
looks better

Reply:
yes, it look better this way.
thanks @oshtaier for your comments.